chatgpt/chatgpt.py

```diff
@@ -289,7 +289,9 @@ def load_config(self, config_path: str = DEFAULT_CONFIG_PATH):
             raise ChatgptError("Error loading the configuration file in \"{}\"".format(self._config_path),
                                ChatgptErrorCodes.CONFIG_FILE_ERROR) from e
 
-    def write_config(self, config_path: str = DEFAULT_CONFIG_PATH):
+    def write_config(self, config_path: Optional[str] = None):
```

Comment:
Creating Conversation by only `email` and `password`, will cause `self._config_path` is None.
Then calling `chat()`, the `chat` will call `self.write_config(self._config_path)` and it is actually `self.write_config(None)`.

So, the default value will not work in this situation.